storage/Utils/ColumnConfigFile.h

```diff
@@ -118,6 +118,9 @@ class ColumnConfigFile : public CommentedConfigFile
 
 
     ColumnConfigFile();
+
+    ColumnConfigFile(int permissions);
```

Comment:
We don't need another constructor for that. Simply provide a default value (say, 0644) for the _permissions_ argument, then we also have a parameter-less default constructor as a byproduct.

That avoids the duplication that we now have.